src/main/java/org/broadinstitute/hellbender/tools/walkers/longreads/RecoverUncorrectedReads.java

```diff
@@ -0,0 +1,116 @@
+package org.broadinstitute.hellbender.tools.walkers.longreads;
+
+import htsjdk.samtools.*;
+import org.broadinstitute.barclay.argparser.Argument;
+import org.broadinstitute.barclay.argparser.BetaFeature;
+import org.broadinstitute.barclay.argparser.CommandLineProgramProperties;
+import org.broadinstitute.barclay.argparser.ExperimentalFeature;
+import org.broadinstitute.barclay.help.DocumentedFeature;
+import org.broadinstitute.hellbender.cmdline.StandardArgumentDefinitions;
+import org.broadinstitute.hellbender.engine.FeatureContext;
+import org.broadinstitute.hellbender.engine.GATKPath;
+import org.broadinstitute.hellbender.engine.ReadWalker;
+import org.broadinstitute.hellbender.engine.ReferenceContext;
+import org.broadinstitute.hellbender.exceptions.UserException;
+import org.broadinstitute.hellbender.utils.io.IOUtils;
+import org.broadinstitute.hellbender.utils.read.GATKRead;
+import org.broadinstitute.hellbender.utils.read.SAMFileGATKReadWriter;
+import picard.cmdline.programgroups.ReadDataManipulationProgramGroup;
+
+import java.io.IOException;
+import java.util.HashSet;
+import java.util.Set;
+
+/**
+ * Recover uncorrected reads
+ *
+ * <h3>Input</h3>
+ * <ul>
+ *     <li>The uncorrected BAM file</li>
+ *     <li>The CCS-corrected BAM file</li>
+ * </ul>
+ *
+ * <h3>Outputs</h3>
+ * <ul>
+ *     <li>A BAM file of remaining subreads that were not corrected by the CCS step</li>
+ * </ul>
+ *
+ * <h3>Usage Example</h3>
+ * <h4>Quickly count errors</h4>
+ * <pre>
+ *   gatk RecoverUncorrectedReads \
+ *     -I input.bam \
+ *     -C ccs.bam \
+ *     -O output.bam
+ * </pre>
+ */
+@DocumentedFeature
+@ExperimentalFeature
+@CommandLineProgramProperties(
+        summary = "Recover uncorrected reads",
+        oneLineSummary = "Recover uncorrected reads",
+        programGroup = ReadDataManipulationProgramGroup.class
```

Comment:
this goes in the long reads program group and possibly needs another few woords of summary